waiting_room.go

```diff
@@ -532,3 +550,56 @@ func (api *API) DeleteWaitingRoomRule(ctx context.Context, rc *ResourceContainer
 
 	return r.Result, nil
 }
+
+// WaitingRoomSettings fetches the Waiting Room zone-level settings for a zone.
+//
+// API reference: https://api.cloudflare.com/#waiting-room-get-zone-settings
+func (api *API) WaitingRoomSettings(ctx context.Context, zoneID string) (WaitingRoomSettings, error) {
```

Comment:
here is the example from my comment in the PR review.

```suggestion
func (api *API) GetWaitingRoomSettings(ctx context.Context, *rc ResourceContainer, params GetWaitingRoomSettingsParams) (WaitingRoomSettings, error) {
```

even if you don't need the `GetWaitingRoomSettingsParams` right now, we'll add it to match the conventions and make it easy to extend in the future.